### PR TITLE
ILS / LOC Fixes

### LINKED ISSUE #2306: OEAB
# Airport ICAO
OEAB

# Runway(s) with ILS
13
# References (Official sources/charts)
http://www.airmate.aero/php/pdf_viewer/web/viewer.php?pdf=OE/OEAB/400000000365179.pdf 

# Are you working on this?

Yes

### LINKED ISSUE #2313: LTAJ
# Airport ICAO
LTAJ

# Runway(s) with ILS
28
# References (Official sources/charts)
http://www.airmate.aero/php/pdf_viewer/web/viewer.php?pdf=LT/LTAJ/400000000533871.pdf 

# Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by SuhasIF**
OEAB - (Fixed #2306)
LTAJ - (Fixed #2313)

### CODE REVIEW

**Comment by SuhasIF** [comment]
I'd like this to be merged, please. Thanks!

File: Glideslope.json [hunk 1]
```diff
@@ -30589,6 +30589,19 @@
...
+   "name": "GS",
+   "airportICAO": "LTAJ",
+   "identifier": "IGNP",
+   "associatedRunwayNumber": "28",
+   "frequency": 109100,
+   "elevation": 2305,
+   "latitude": 36.9446004,
+   "longitude": 37.49463091,
+   "bearing": 285.19,
+   "glideslope": 3.00028519,
```
Comment (velocity23) [inline Glideslope.json:30602]: Should be 3.0028519 (heading is 285.19 not 28.519)

**Review by velocity23 — CHANGES_REQUESTED** [review]

**Comment by velocity23** [comment]
@SuhasIF are you planning to fix this?

**Comment by SuhasIF** [comment]
@velocity23 fixed. Apologies for the delay!

**Pull request merged**